hydra-access-controls/spec/unit/permissions_spec.rb

```diff
@@ -50,6 +50,19 @@ class Foo < ActiveFedora::Base
         subject.save!
         subject.permissions_attributes = [{ type: "person", access: "edit", name: "jcoyne" }]
       end
+
+      context "with ids" do
+        let(:permission_id) { ActiveFedora::Base.uri_to_id(subject.permissions.last.rdf_subject.to_s) }
+        before do
+          # Load up a cached copy
+          subject.permissions.map(&:to_hash)
+          subject.permissions_attributes = [{ id: permission_id, type: "person", access: "read", name: "jcoyne" }]
+        end
+        it "clears the cache" do
+          expect(subject.permissions.map(&:to_hash)).to eq [{ type: "person", access: "read", name: "jcoyne" }]
```

Comment:
I'm not following how this spec clears the cache?

I see line 58 warms the cache, then line 59 changes the permissions. And line 62 reflects the old cache value. Regarding cache clearing I would've expected a spec that leveraged `expect {}.to change {}.from().to()`.